recipes/boost/all/conanfile.py

```diff
@@ -8,20 +8,46 @@
 import sys
 import shlex
 import shutil
+import yaml
 
 try:
     from cStringIO import StringIO
 except ImportError:
     from io import StringIO
 
-# From from *1 (see below, b2 --show-libraries), also ordered following linkage order
-# see https://github.com/Kitware/CMake/blob/master/Modules/FindBoost.cmake to know the order
-
-lib_list = ['math', 'wave', 'container', 'contract', 'exception', 'graph', 'iostreams', 'locale', 'log',
-            'program_options', 'random', 'regex', 'mpi', 'serialization',
-            'coroutine', 'fiber', 'context', 'timer', 'thread', 'chrono', 'date_time',
```

Comment:
The "context" library seems to have disappeared from all the lists, breaking the once-existing `without_context` option. Was this intentional?

Reply:
No it was not.
FYI, `context` has to be added to `conanfile.py` + `rebuild-dependencies.py`.